x-pack/plugins/serverless_search/public/application/components/overview.tsx

```diff
@@ -309,6 +309,7 @@ export const ElasticsearchOverview = () => {
           docLinks={docLinks}
           application={application}
           sharePlugin={share}
+          http={http}
```

Comment:
Added this to redirect to create connector flow  `'/app/connectors'` in `ingestions_panel`